Libplanet/Net/Swarm.cs

```diff
@@ -1891,6 +1891,16 @@ await foreach (Block<T> block in blocks)
                             block.Hash
                         );
                     }
+
+                    var getBlockListCount = await blocks.CountAsync(cancellationToken);
+                    if (getBlockListCount < FindNextHashesChunkSize)
+                    {
+                        _logger.Debug(
+                            $"Get all blocks from Peer [{peer}]",
```

Comment:
```suggestion
                            $"Got all blocks from Peer [{peer}]",
```
And how about log block count here?

Reply:
amended at https://github.com/planetarium/libplanet/pull/1112/commits/969588c050c433949763a7ec7fee88f9f25e7cfa. I added log when adding test code.